dipy/reconst/csdeconv.py

```diff
@@ -775,8 +809,9 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7,
         radius of cubic ROI
     fa_thr : float
         FA threshold
-    fa_operator : lambda
-        operator used to compare the FA with the fa_thr.
+    fa_operator : function
+        Function that defines operator used to compare the FA with the fa_thr.
+        Must be defined as `def function_name(FA, fa_thr)` and return a bool array.
```

Comment:
I think that the code is fine (I think it's easier to understand this way. The documentation can be changed to say:

    fa_operator : callable
        A callable that defines an operation that compares FA with the fa_thr. The operator 
        should have two positional arguments (e.g., `fa_operator(FA, fa_thr)`) and it should
        return a bool array.